index.html

```diff
@@ -27,40 +27,40 @@
 <head>
 
 	<!-- TO TRANSLATE: "title", "description" -->
-	<title>The Evolution of Trust</title>
-	<meta name="description" content="an interactive guide to the game theory of why &amp; how we trust each other"/>
+	<title>Ewolucja Zaufania</title>
+	<meta name="description" content="Interaktywny przewodnik po zaufaniu według teorii gier"/>
```

Comment:
OK, ja się już wdałam w kłótnię z koleżanką korektorką, która twierdzi, że da się oprowadzać po zaufaniu, i że to jest zrozumiałe :D 

Proponuję poza mechanizmami:
praktyki
schematy
zasady
systemy
architektura
sieć

Kumpela proponuje:
fundamenty
reguły

ale to się rozmija znaczeniowo, jeśli nic nie pasi to wróćmy do oprowadzania po zaufaniu, przynajmniej będzie poezja